src/main/resources/com/cloudbees/hudson/plugins/folder/AbstractFolder/configure.jelly

```diff
@@ -66,6 +66,9 @@ THE SOFTWARE.
 
         <div class="jenkins-!-margin-top-6">
           <f:entry title="${%Display Name}" field="displayNameOrNull">
+            <f:invisibleEntry>
+              <f:textbox name="jobName" value="${it.name}"/>
+            </f:invisibleEntry>
```

Comment:
I doubt you need this, or the `checkDependsOn`. (Or for that matter the `checkUrl`, provided the web method has the appropriate name: `doCheckDisplayNameOrNull`) Try replacing `@QueryParameter String jobName` with `@AncestorInPath AbstractFolder folder`.

Reply:
Yes you're right AncestorInPath works well,  the only thing it's a little bit weird that we need empty `checkDependsOn=""`. Applied in [a3e7976](https://github.com/jenkinsci/cloudbees-folder-plugin/pull/268/commits/a3e7976633175a326562274397611b8dc7b91ee0)